staging/src/k8s.io/apiserver/pkg/endpoints/handlers/patch.go

```diff
@@ -446,28 +449,94 @@ func (p *patcher) applyAdmission(ctx context.Context, patchedObject runtime.Obje
 	return patchedObject, nil
 }
 
+func noOpTransform(_ context.Context, newObj runtime.Object, _ runtime.Object) (runtime.Object, error) {
+	return newObj, nil
+}
+
+type convertor func(runtime.Object) (runtime.Object, error)
+
+func newConvertor(objectConvertor runtime.ObjectConvertor, versioner runtime.GroupVersioner) convertor {
+	return func(obj runtime.Object) (runtime.Object, error) {
+		return objectConvertor.ConvertToVersion(obj, versioner)
+	}
+}
+
+// TODO(kwiesmueller): typeConverter & convertor naming needs "fixing"
```

Comment:
what prevents you from fixing the name now?

Reply:
It was meant as a todo while creating the PR. So nothing long term I just wanted to move it until i finished what I was doing back then.

And I got to think about it a little more/understand why convertor is named convertor etc. So I'm on it :+1: